backend/src/test/java/com/redhat/cloud/notifications/routers/NotificationResourceTest.java

```diff
@@ -880,17 +885,23 @@ void testInsufficientPrivileges() {
                 .then()
                 .statusCode(403);
 
+        final CreateBehaviorGroupRequest createBehaviorGroupRequest = new CreateBehaviorGroupRequest();
+        createBehaviorGroupRequest.bundleId = UUID.randomUUID();
+        createBehaviorGroupRequest.displayName = "create-behavior-group-test";
         given()
                 .header(readAccessIdentityHeader)
                 .contentType(JSON)
+                .body(this.objectMapper.writeValueAsString(createBehaviorGroupRequest))
```

Comment:
Body is not requested to check 403.
Then we can validate checks order: first secutity, then content format.